avm/res/app/container-app/main.bicep

```
@@ -217,6 +220,7 @@ resource containerApp 'Microsoft.App/containerApps@2024-03-01' = {
       ingress: disableIngress
         ? null
         : {
+            additionalPortMappings: !empty(additionalPortMappings) ? additionalPortMappings : null
```

```suggestion
            additionalPortMappings: additionalPortMappings
```
As it is nullable, the default would anyways be `null`

---

Ah, good point.